### DISCUSSION TITLE
the time ticks of waterfall plot does not zoom automatically

### DISCUSSION THREAD

**Discussion by jinwar** [DASDAE/dascore#148] [Ideas]
![image](https://github.com/DASDAE/dascore/assets/2571950/421be474-7084-4e32-910a-ece3858d6569)

**Comment by d-chambers** [comment]
closed by #156

**Comment by d-chambers** [comment]
@jinwar, If you get a few minutes to try out the new time formatting on the master branch's waterfall plot let me know what you think. We can easily make changes before it lands in a release.
